### DISCUSSION TITLE
only bg-black is working

### DISCUSSION THREAD

**Discussion by azr-arch** [tailwindlabs/tailwindcss#10686] [Help]
i am trying to use tailwindcss in a simple html,css,js project, but only bg-black is working not other colors

**Comment by wongjn** [comment]
Consider checking:

- Tailwind has run a compile since the last time you have made changes.
- The `content` file globs sufficiently cover the source files that use the class names.
- Class names are not built with string concatenation/interpolation.

Otherwise, try recreating the unexpected behavior in a [Tailwind Play](https://play.tailwindcss.com). If it works in Tailwind Play, then this may indicate some misconfiguration. In which case, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
